sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Models/TimeSeries2Id.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    /// <summary>
+    /// A single Time Series Id value that is an array of 2 string values that uniquely identifies a Time Series instance.
```

Comment:
Would be good to add description on what this means or in other words something like "Value for TSI Id is chosen/given when creating a Azure Time Series Insights Gen2 environment through Azure portal"